src/utils/apolloClient.ts

```diff
@@ -4,9 +4,7 @@ import { Cookies } from 'react-cookie-consent'
 
 // needed for CORS, see https://www.apollographql.com/docs/react/networking/authentication/#cookie
 const httpLink = createHttpLink({
-  uri:
-    (process.env.NEXT_PUBLIC_API_URL || 'https://api.stage.datacite.org') +
-    '/graphql',
+  uri: (operation) => `${(process.env.NEXT_PUBLIC_API_URL || 'https://api.stage.datacite.org')}/graphql?graphqlOperationName=${encodeURIComponent(operation.operationName)}`,
```

Comment:
Looks good to me. One thing I noticed, the current names for the graphQl queries aren't very granular, for instance we are using a query called `getOrganizationQuery` 8 different times across the DOI, organization, and API pages. We may need to update the query names to make the analysis more effective

Reply:
Thats a great point @bklaing2 I will change the query names too.

Reply:
I will create new PR to address the query rename changes.